src/Service/Caller/Masa.Contrib.Service.Caller/DefaultRequestIdGenerator.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Service.Caller;
+
+public class DefaultRequestIdGenerator : IRequestIdGenerator
+{
+    public string NewId() => Guid.NewGuid().ToString();
```

Comment:
Use IdGenerator

Reply:
Using IdGenerator requires additional reference to the package of IdGenerator, and it is troublesome to change the way to generate id

Recommended Use

services.Configure<CallerFactoryOptions>(option => 
{
    option.IdGeneratorFunc = serviceProvider => serviceProvider.GetRequiredService<IGuidGenerator>().NewId().ToString();
})